### ISSUE TITLE
Hoverover timeline causes reflow as scrollbar unexpectedly pops up 

### ISSUE THREAD

**Issue by ara4n** [vector-im/riot-web#12446]
FF release on macOS; might be related to compact theme or custom CSS?

https://mozilla.modular.im/_matrix/media/r0/download/mozilla.modular.im/e1d1dadee8c3e3c3334870eb1365a69a64512ad0

**Comment by bwindels** [comment]
This is the AutoHideScrollbar CSS not working properly. It adds negative margin to the right to compensate for the scrollbar width when setting `overflow: scroll` on hover. It's working for me on Linux, maybe somebody on mac can have a look why it's not behaving in the same way?

![togglehover](https://user-images.githubusercontent.com/274386/75017619-4b5d2300-5485-11ea-8e68-1f551f9b14ff.gif)

**Comment by jryans** [comment]
Hmm, I can't see to reproduce this on macOS either... I've tried both the light and dark Riot themes. I've also tried playing around the different scrollbar settings in macOS system settings, but no luck there.

@Pike, does this happen for you in all rooms? Does it happen the room timeline is scrolled to any position? Does it happen in clean Firefox profile?

So far, this is only report of this issue, so my suspicious is there's something unusual happening in your local config that we need to discover first.
